nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -246,17 +246,51 @@ define([
                 },
             }),
             state = getInitialState(),
-            specs = {},
-            rawSpecs = model.getItem('app.specs');
+            // These are the processed Spec object with proper layout order, etc.
+            specs = {};
 
-        Object.keys(rawSpecs).forEach((appId) => {
+        for (const [appId, appSpec] of Object.entries(model.getItem('app.specs'))) {
```

Comment:
This is the only place that `rawSpecs` is used, though it's likely kept in memory, so I refactored this a touch.